pages/Filter/components/PriceRangeSlider/index.jsx

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import isEqual from 'lodash/isEqual';
+/** @type {{currency: string}} */
```

Comment:
Please remove this line. We don't have this anywhere. No reason to start it now.

Reply:
+